R/pkg/inst/tests/testthat/test_mllib_clustering.R

```diff
@@ -99,7 +99,8 @@ test_that("spark.kmeans", {
 
   take(training, 1)
 
-  model <- spark.kmeans(data = training, ~ ., k = 2, maxIter = 10, initMode = "random")
+  model <- spark.kmeans(data = training, ~ ., k = 2, maxIter = 10, initMode = "random", seed = 1,
+                        initSteps = 3, tol = 1E-5)
```

Comment:
It looks like the test case is insensitive to ```seed```, could you add a test case which is sensitive to ```seed```? Or add a test case whose termination controlled by ```tol```? We need to make sure these arguments will take effect.